### DISCUSSION TITLE
有一个叫DB-GPT的项目和本项目类似，作者如有空可以了解一下

### DISCUSSION THREAD

**Discussion by shmily91** [wenda-LLM/wenda#298] [General]
这个项目有几个亮点的地方作者可借鉴一下：
1、SQL 语言能力
2、私域问答与数据处理
3、插件模型
支持自定义插件执行任务，原生支持Auto-GPT插件。如:
SQL自动执行，获取查询结果
我认为最大的亮点在于：自动爬取学习知识。
供作者参考

**Comment by l15y** [comment]
没有在示例里看到有相关自动爬取的内容，不过估计应该和bing sogo模式下的网页全文抓取类似。看issue里的意思，应该是todo
数据库做知识库挺有意思，相当于另一种形式的知识图谱，不过目前没有这方面的数据源，没有需求也无法测试

**Comment by l15y** [comment]
他的插件模型似乎比我的要简单的多，但是说支持autogpt，这就是另一个方向了，对模型智力要求比较高，后期也会考虑
